job-search-web-scraping.py

```diff
@@ -19,16 +19,26 @@ def indeed_job_search():
     browser.implicitly_wait(5) 
 
     search_results = browser.find_elements_by_xpath('//h2/a')
-
+    search_descriptions=browser.find_elements_by_xpath('//div[@class="summary"]')
+    
+    descriptions=[]
+    for element in search_descriptions:
+        job_description=element.text
+        descriptions.append(job_description)
+        
     file = open("job_search.txt", 'a')
     file.write("\n")
-
+    
+    index=0
     for job_element in search_results:
 
         job_title = job_element.text
         job_link = job_element.get_attribute('href')
+        
+
+        file.write("%s | link: %s | description: %s \n" %(job_title, job_link, descriptions[index]))
```

Comment:
Can you refactor to make this section one for loop instead?

Reply:
Yes I will make the changes, and can you tell how to pull the latest upstream? I'm new to open source and don't have much idea on it

Reply:
@btapash Sure, just follow these steps:

add link to upstream in your local repo
```
git remote add upstream https://github.com/inspirezonetech/JobSearchWebScraping.git
```

you can confirm the upstream was added by running
```
git remote -v
```

pull main branch from the upstream to update your local repo
```
git pull upstream main
```

now make your changes on your branch and submit the pull request

Reply:
Hi, Any updates?

Reply:
There is some problem with my laptop, it is in service centre for repairing. I can do it after a week or you can assign someone else.

Reply:
Okay no problem thanks for letting me know. I'll unassign you to leave it open. If its still open when your laptop is fixed you can try again.